daemon/plugin-fuzz/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "ethersync-plugin-fuzz"
```

Comment:
The new crate does two things – run the old fuzzer (an "overall randomized integration test"), and test the Neovim plugin for correct behavior (which is not fuzzer-like at all).

Could we find a better name for the crate, that's more self-descriptive? Maybe "ethersync-integration-tests"? Then we could also call the directory "integration-tests", making it easy to see what's inside.

Reply:
This makes perfect sense! Done on both accounts.